samples/apps/copilot-chat-app/webapi/Config/DocumentMemoryOptions.cs

```diff
@@ -46,4 +46,12 @@ public class DocumentMemoryOptions
     /// </summary>
     [Range(0, int.MaxValue)]
     public int FileSizeLimit { get; set; } = 1000000;
+
+    /// <summary>
+    /// Similarity threshold to avoid document memory duplication when importing documents to memory.
+    /// The higher the value, the more similar document memories will be allowed.
```

Comment:
That comment makes my head spin.

Is it:
The higher the value, the more documents have to be similar to be eliminated as duplicates
OR
The higher the value, the more documents can be similar without being eliminated as duplicates

Reply:
The second one

Reply:
Yeah, maybe replace the two lines of the comment above (and in the settings) with the second line above.

Reply:
To me, this single line "The higher the value, the more documents can be similar without being eliminated as duplicates" is way easier to understand than the 2 that are currently there.  Unless you have a reason not to, just use that one line